data_steward/cdr_cleaner/cleaning_rules/clean_mapping.py

```diff
@@ -47,6 +47,13 @@
 ISSUE_NUMBER = 'DC-715'
 
 
+def get_mapping_tables():
+    """
+    Returns list of mapping tables in fields path
+    """
+    return resources.MAPPING_TABLES
```

Comment:
I think this may need to return a list of `_mapping_` and `_ext` tables for the reporting.  I checked the `generate_ext_tables.py` file.  It basically just creates an extension table for any mapping tables.  You could use a list comprehension to do the same thing here with this `resources.MAPPING_TABLE` variable as your input.